ansible/roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
this should be rather `roles/service-user`, no?

Reply:
It's not a bad idea tho, but I just used `main.yaml` since we do not have multiple files in the directory